devise.rb

```diff
@@ -43,6 +43,11 @@
   gsub_file('app/views/layouts/application.html.erb', "<%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>", scripts)
 end
 gsub_file('app/views/layouts/application.html.erb', "<%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>", "<%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>")
+style = <<~HTML
+  <meta name=“viewport” content=“width=device-width, initial-scale=1, shrink-to-fit=no”>
+      <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+HTML
+gsub_file('app/views/layouts/application.html.erb', "<%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>", style)
```

Comment:
Nice hack, you can (should?) use [`inject_into_file "", before: XXX`](https://github.com/lewagon/rails-templates/blob/rails-six/devise.rb#L5)

Reply:
I tried this at the beginning but it broke indentation into `application.html.erb`